organization/governance-model.md

```diff
@@ -10,16 +10,18 @@ This provides a non-legal summary of the Foundation's governance model as establ
 
 ## Overview
 
-The Foundation for Public Code is a [member-owned association (vereniging) under Dutch law](https://business.gov.nl/starting-your-business/choosing-a-business-structure/association/). Only wholly publicly owned organizations may become members.
+The Foundation for Public Code is a [non-profit association (vereniging) under Dutch law](https://business.gov.nl/starting-your-business/choosing-a-business-structure/association/).
 
-This means members are responsible for and steer the Foundation's operations, including priorities and annual budget for the organization.
+The association's members are the regional Foundation for Public Code chapter organizations.
+These members are responsible for and steer the association's operations, including priorities and annual budget for the organization.
 
 ## Foundation bodies
 
 The Foundation consists of four bodies:
```

Comment:
```suggestion
The Foundation consists of five bodies:
```
With the addition of the strategic council to the list, we should +1 here.

Reply:
(I am assuming the resolve without comment was a miss-click.)

Reply:
It's fixed!